src/MarginTrading.Backend/Controllers/ScheduleSettingsController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using JetBrains.Annotations;
+using MarginTrading.Backend.Contracts.DayOffSettings;
+using MarginTrading.Backend.Core.DayOffSettings;
+using MarginTrading.Backend.Services.AssetPairs;
+using Microsoft.AspNetCore.Mvc;
+
+namespace MarginTrading.Backend.Controllers
+{
+    [Route("api/[controller]")]
+    public class ScheduleSettingsController : Controller
```

Comment:
Don't understand why you created 2 controllers for 1 business process and working with 1 service.
As for me it would be more clear if we will have 1 controller with routes api/ScheduleSettings/root and api/ScheduleSettings/Exclusions

Reply:
My point of view was "two different entities"